SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -594,6 +595,12 @@ void UGlobalStateManager::BeginDestroy()
 
 void UGlobalStateManager::BecomeAuthoritativeOverAllActors()
 {
+	// This logic is not used in offloading.
+	if (USpatialStatics::IsSpatialOffloadingEnabled())
```

Comment:
Add to `BecomeAuthoritativeOverActorsBasedOnLBStrategy` also?

Reply:
`BecomeAuthoritativeOverActorsBasedOnLBStrategy` is only called when the Unreal Load Balancer is enabled, so offloading cannot be enabled at the same time.

Reply:
Defensive coding!